src/pages/admin/quizzes/QuizEdit/index.js

```diff
@@ -1,30 +1,34 @@
 import React, { Fragment, useState, useEffect } from 'react';
 import { Link, useParams } from 'react-router-dom';
 import { useToast } from '../../../../hooks/useToast';
-import Button from 'react-bootstrap/Button';
+import Button from '../../../../components/Button';
 import Nav from 'react-bootstrap/Nav';
 import Spinner from 'react-bootstrap/Spinner';
 import { AiOutlineCloseCircle } from 'react-icons/ai';
 
 import style from './index.module.scss';
 import QuestionType from './components/QuestionType';
-// import ChangeLocation from '../../../../components/ChangeLocation';
+import ChangeLocation from '../../../../components/ChangeLocation';
 
 import QuizApi from '../../../../api/Quiz';
 import QuestionApi from '../../../../api/Question';
+import CategoryApi from '../../../../api/Category';
 
 const QuizEdit = () => {
-  // const TYPE = 'withPathDisplay';
-  // const [show, setShow] = useState(false);
-  // const handleClose = () => setShow(false);
-  // const handleShow = () => setShow(true);
-  // const [location, setLocation] = useState(null);
-  // const [locationPathDisplay, setLocationPathDisplay] = useState('');
+  const TYPE = 'withPathDisplay';
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+  const [location, setLocation] = useState(null);
+  const [locationPathDisplay, setLocationPathDisplay] = useState('');
   const [quizInfo, setQuizInfo] = useState(null);
   const { categoryId, quizId } = useParams();
   const [questions, setQuestions] = useState(null);
   const [selectedQuestion, setSelectedQuestion] = useState(null);
   const toast = useToast();
+  const [changeCategoryId, setChangeCategoryId] = useState(null);
+  const [parentCategories, setParentCategories] = useState(null);
+  const [saveLocation, setSaveLocation] = useState(false);
```

Comment:
please group together the variables. for example group together the ones using useState, not using useState, etc..